packages/webapp/pages/onboarding.tsx

```diff
@@ -110,7 +123,15 @@ export function OnboardPage(): ReactElement {
   const targetId: string = ExperimentWinner.OnboardingV4;
   const formRef = useRef<HTMLFormElement>();
   const [activeScreen, setActiveScreen] = useState(OnboardingStep.Intro);
-  const enableContentTypeStep = useRef(false);
+  const { updateAdvancedSettings } = useMutateFilters(user);
+  const shouldUpdateAdvancedSettings = useRef(true);
+  const isSeniorUser =
+    EXPERIENCE_TO_SENIORITY[user?.experienceLevel] === 'senior' ||
+    user?.experienceLevel === 'MORE_THAN_4_YEARS';
```

Comment:
Why doesn't `EXPERIENCE_TO_SENIORITY` do both cases?

Reply:
Cuz this is not senior 'MORE_THAN_4_YEARS' in that map, and product wanted also this category for experiment. 

We already user EXPERIENCE_TO_SENIORITY in some logs and didn't want to mess with that. So I think after the experiment results we can see what we want to do overall.